feedback-module/src/styles/_components.scss

```diff
@@ -0,0 +1,136 @@
+  @extend .mobile-lg-padding-x-205,
+    .mobile-lg-padding-y-105,
+    .mobile-lg-font-sans-md2,
+    .mobile-lg-margin-y-105 !optional;
+  @extend .mobile-lg\:padding-x-205,
+    .mobile-lg\:padding-y-105,
+    .mobile-lg\:font-sans-md2,
+    .mobile-lg\:margin-y-105 !optional;
+  &:hover {
+    @extend .bg-primary-light,
+      .text-primary,
+      .border-primary,
+      .border-bottom-2px;
+  }
+}
+
+// Checkbox
+.usa-checkbox {
+  @extend .text-bold,
+    .text-primary,
+    .bg-transparent,
+    .font-sans-md,
+    .margin-y-1,
+    .text-no-uppercase;
+  @extend .mobile-lg\:font-sans-lg, .mobile-lg\:margin-y-105 !optional;
+  @extend .mobile-lg-font-sans-lg, .mobile-lg-margin-y-105 !optional;
+
+  &__label {
+    @extend .text-bold;
+  }
+}
+
+.usa-label {
+  @extend .text-primary,
+    .text-bold,
+    .font-sans-md2,
+    .line-height-sans-2,
+    .maxw-none,
+    .margin-y-205,
+    .text-no-uppercase;
+  @extend .mobile-lg-font-sans-lg2 !optional;
+  @extend .mobile-lg\:font-sans-lg2 !optional;
+}
+
+.usa-input,
+.usa-textarea,
+.usa-checkbox__label {
+  @extend .text-primary, .font-sans-md;
+  @extend .mobile-lg\:font-sans-lg !optional;
+  @extend .mobile-lg-font-sans-lg !optional;
+}
+
+.usa-textarea,
+.usa-input {
+  @extend .text-primary, .border-05, .border-primary, .bg-primary-light;
+  &:focus {
+    @extend .border-width-6px, .bg-primary-lighter;
+  }
+}
+
+.usa-input {
+  @extend .height-auto, .flex-1;
+}
+
+.usa-form {
+  @extend .maxw-none,
+    .width-full,
+    .display-flex,
+    .flex-column,
+    .flex-align-stretch;
+}
+
+.usa-alert__text {
+  @extend .display-flex;
+}
+
+.feedback-header {
+  @extend .radius-top-lg, .bg-primary, .display-flex, .padding-1;
+  @extend .mobile-lg\:padding-x-2, .mobile-lg\:padding-y-105 !optional;
+  @extend .mobile-lg-padding-x-2, .mobile-lg-padding-y-105 !optional;
+}
+
+.feedback-module-container {
+  @extend .bg-primary-light, .radius-top-lg, .padding-x-0, .shadow-btn;
+  @extend .mobile-lg\:width-desktop-sm !optional;
+  @extend .mobile-lg-width-desktop-sm !optional;
+}
+
+.feedback-screen-container {
+  @extend .padding-x-5, .padding-y-205;
+  @extend .mobile-lg\:padding-x-7, .mobile-lg\:padding-y-6 !optional;
+  @extend .mobile-lg-padding-x-7, .mobile-lg-padding-y-6 !optional;
+}
+
+.feedback-h1 {
+  @extend .text-bold,
+    .text-left,
+    .margin-top-0,
+    .margin-bottom-3,
+    .text-no-uppercase,
+    .line-height-sans-2;
+  @extend .mobile-lg-font-sans-lg2 !optional;
+  @extend .mobile-lg\:font-sans-lg2 !optional;
+
+  &--dark {
+    @extend .text-primary;
+  }
+  &--light {
+    @extend .text-white;
+  }
+}
+
+.feedback-plaintext {
+  @extend .font-sans-md,
+    .line-height-sans-3,
+    .text-primary,
+    .margin-0,
+    .padding-y-3,
+    .text-no-uppercase;
+  @extend .mobile-lg-font-sans-lg, .mobile-lg-line-height-sans-5 !optional;
+  @extend .mobile-lg\:font-sans-lg, .mobile-lg\:line-height-sans-5 !optional;
+
+  p {
+    @extend .margin-0;
+  }
+}
```

Comment:
If two classes share a lot of styling in common can one of them extend the other so we don't have so much repetition within this file?

Reply:
Hmmm the thing with that is that if we have one extend the other it would extend _all_ of the styling from the other (if we're also considering the uswds classes, it'd also extend the default uswds styling for one of those classes as well, that we didn't set). I think maybe an option to handle the common styling would be to use grouping selectors and move all the common styles in there?

For example
```
.usa-label, .usa-checkbox, .feedback-h1, etc {
@extend .text-bold, .text-primary;
}
```

What do you think?

Reply:
Yes I think that makes sense, I think we should do that